docs/get_started/install.md

```diff
@@ -424,13 +443,214 @@ $ sudo python setup.py install
 </div>
 <!-- END - Linux Python GPU Installation Instructions -->
 
-<div class="linux macos">
+<!-- START - MacOS Python CPU Installation Instructions -->
 
-# Validate MXNet Installation
+<div class="macos">
+  <div class="python">
+    <div class="cpu">
+
+The following installation instructions have been tested on OSX Sierra and El Capitan.
+
+
+**Prerequisites**
+
+If not already installed, [download and install Xcode](https://developer.apple.com/xcode/) for macOS. [Xcode](https://en.wikipedia.org/wiki/Xcode) is an integrated development environment for macOS containing a suite of software development tools like C/C++ compilers, BLAS library and more.
+
+<div class="virtualenv">
+<br/>
+
+**Step 1**  Install prerequisites - Homebrew, python development tools.
+
+```bash
+# Install Homebrew
+$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+$ export PATH=/usr/local/bin:/usr/local/sbin:$PATH
+
+# Install python development tools - python2.7, pip, python-setuptools
+$ brew install python
+```
+
+**Step 2**  Install virtualenv for macOS.
+
+```bash
+$ pip install virtualenv
+```
+
+**Step 3**  Create and activate virtualenv environment for MXNet.
+
+Following command creates a virtualenv environment at `~/mxnet` directory. However, you can choose any directory by replacing `~/mxnet` with a directory of your choice.
+
+```bash
+$ virtualenv --system-site-packages ~/mxnet
+```
+
+Activate the virtualenv environment created for *MXNet*.
+
+```bash
+$ source ~/mxnet/bin/activate
+```
+
+After activating the environment, you should see the prompt as below.
+
+```bash
+(mxnet)$
+```
+
+**Step 4**  Install MXNet in the active virtualenv environment.
+
+Installing *MXNet* with pip requires a latest version of `pip`. Install the latest version of `pip` by issuing the following command.
+
+```bash
+(mxnet)$ pip install --upgrade pip
+(mxnet)$ pip install --upgrade setuptools
+```
+
+Install *MXNet* with OpenBLAS acceleration.
+
+```bash
+(mxnet)$ pip install mxnet
+```
+
+**Step 5**  Validate the installation by running simple *MXNet* code described [here](#validate-mxnet-installation).
+
+**Note**  You can read more about virtualenv [here](https://virtualenv.pypa.io/en/stable/userguide/).
```

Comment:
I agree for now, we have only mxnet, but, I see other dependencies like jupyter, graphviz etc.. will be required. My idea is to evolve this to have a requirement file, which user can use in virtualenv to get complex mxnet environment setup in one command.

Also, for user, we are showing options in buttons, so should not complicate with length.